packages/devtools_app/lib/src/memory/memory_chart.dart

```diff
@@ -204,31 +206,21 @@ class MemoryChartState extends State<MemoryChart> with AutoDisposeMixin {
 
     _timelineSlider = _createTimelineSlider();
 
-    return Row(
+    return Column(
       children: [
-        Expanded(
-          child: memoryTimeline.liveData.isEmpty
-              ? const SizedBox()
-              : Column(
-                  mainAxisSize: MainAxisSize.min,
-                  children: [
-                    SizedBox(
-                      height: defaultChartHeight,
-                      child: LineChart(dartChartController),
-                    ),
-                    _timelineSlider,
-                  ],
-                ),
-        ),
-        if (controller.isAndroidChartVisible)
-          const SizedBox(width: denseSpacing),
-        if (controller.isAndroidChartVisible)
-          Expanded(
-            child: SizedBox(
-              height: defaultChartHeight,
-              child: LineChart(androidChartController),
-            ),
-          ),
+        memoryTimeline.liveData.isEmpty
+            ? const SizedBox(height: 0)
```

Comment:
nit: is the height: 0 really needed?

Reply:
null implies the height of the box shouldn't be constrained.